build.gradle

```diff
@@ -16,9 +16,17 @@
 
 buildscript {
     ext.versions = [
+        'android'          : [
+            "buildTools": "29.0.3",
```

Comment:
Is there any way to pull these from the android-maps-utils Java library? Keeping these in sync will be a pain.

Reply:
Not that I can think of. As long as we keep the dependencies really light (i.e. `maps-utils-ktx` should only depend on AMU and kotlin-specific libs) keeping them in sync shouldn't be too much of a hassle.

That said, I'll remove `appcompat` as a dependency since it doesn't really need it. Eventually `playServicesMaps` and `kotlinxCoroutines` should be removed too and instead be a dependency on a `maps-ktx` module (i.e. KTX for Maps Core).